### PR TITLE
[Cherry Pick] Update release state for recipe bug fix and dependabot version upgrades for 0.12.1

### PR DESCRIPTION
**Pull request by markurtz**

### CODE REVIEW

File: src/sparsify/version.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 from datetime import date
 
 
-version_base = "0.12.0"
+version_base = "0.13.0"
```
Comment (markurtz) [inline src/sparsify/version.py:22]:
```suggestion
version_base = "0.12.1"
```

**Review by bfineran — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sparsify",
-  "version": "0.12.0",
+  "version": "0.13.0",
```
Comment (dhuangnm) [inline package.json:3]: Should the version be 0.12.1 instead?

**Review by dhuangnm — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (markurtz) [reply package.json:3]: ah yep, thanks Dan, misses this one

Comment (markurtz) [inline package.json:3]:
```suggestion
  "version": "0.12.1",
```

**Review by rahul-tuli — DISMISSED** [review]

**Review by bfineran — DISMISSED** [review]

**Review by dhuangnm — DISMISSED** [review]

**Comment by rahul-tuli** [comment]
@markurtz should this be closed?

**Pull request closed without merging**